hoomd/mpcd/test-py/stream_drying_droplet_test.py

```diff
@@ -98,6 +98,8 @@ def test_filler(self):
         drying_sphere = mpcd.stream.drying_droplet(R=self.R1, density = self.density, period=1, boundary="no_slip", seed=1991)
         drying_sphere.set_filler(density=5.0, kT=1.0, seed=79, type='A')
         self.assertTrue(drying_sphere._filler is not None)
+        all_ = hoomd.group.all()
+        mpcd.collide.srd(seed=1221, angle=130., period=1, kT=1.0, group=all_)
```

Comment:
I don't think you need to embed any particles in the collision to test the filler, do you?
```suggestion

        mpcd.collide.srd(seed=1221, angle=130., period=1, kT=1.0)
```

Reply:
Oops, yess this this is not needed